projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.html

```diff
@@ -71,19 +72,21 @@ <h1 class="mb-2" translate>Holdings</h1>
           <ng-container actions *ngIf="last">
             <button type="button" class="btn btn-sm btn-outline-primary ml-1"
                     title="{{ 'Quick receive' | translate }}"
+                    [permissions]="permissions.ITEM_CREATE"
                     (click)="quickIssueReceive()">
-              <i class="fa fa-file-text-o"></i>
+              <i class="fa fa-file-text"></i>
```

Comment:
I think this icon was especially choose by PO, same for `fa-pencil` below (you replaced by `fa-plus`)